testsuite/regressiontests/ticket-396.sli

```diff
@@ -1,123 +0,0 @@
-/*
```

Comment:
Why is this test no longer relevant?

Reply:
Because now the resolution can in fact be changed after the resolution, something this test would detect as wrong behavior. The reason why this is now possible is that we are not actually `finalizing()` and re-`initializing()` each manager upon changing the number of threads anymore, but instead just do whatever is necessary in each manager and thus a previous setting of the resolution survives that operation (which it did not previously).

Reply:
I assume you meant "... now the number of threads can ... be changed after the resolution ..."? Do we have a new test that ensures that changing the number of threads before and after changing the resolution works correct (even in presence of MPI)?

Reply:
I've added a (rather minimal) test in 6e843d4.